metricbeat/module/prometheus/collector/collector.go

```diff
@@ -173,19 +179,15 @@ func (m *MetricSet) skipFamily(family *dto.MetricFamily) bool {
 	// This would mean that we want to keep only the metrics that start with node_ prefix but
 	// are not related to disk so we exclude node_disk_* metrics from them.
 
-	if family == nil {
```

Comment:
Ok, I see what you did here.

I'm not sure if it isn't safer to keep the `family == nil` check here, but split the whole method body into two methods: `skipFamily` and `skipFamilyName`. WDYT? It might prevent next develop